docs/VnVPlan/VnVPlan.tex

```diff
@@ -549,15 +549,330 @@ \subsection{Tests for Nonfunctional Requirements}
+    \item They will ensure that the response provides relevant security headers
+  \end{enumerate}
+
+
+  \item{test-PS-3}
+  
+  NFR: CR-L1, CR-L2, CR-L3, CR-S2, CR-S5
+
+  Type: Manual, Legal Audit
+
+  Initial State: Application ready for use
+
+  Input/Condition: Tester engagement
+  
+  Output/Result: List of vulnerabilites and breach of regulations
+
+  How test will be performed:
+  \begin{enumerate}
+    \item A tester will inspect the code repository and system application against laws and regulations
+    \item All discrepancies between the law and the system will be identified, and updated promptly.
+  \end{enumerate}
+\end{enumerate}
+
+\subsubsection{Development Standards}
+These requirements ensure that the system is built in a way that adheres to
+standard software development principles. This ensures that the code is
+maintainble, scalable, and reliable.
+
+\begin{description}
+  \item[Code Inspection:] The code will be inspected to make sure the system is
+  built adhering to software development standards.
+  \item[Respository Inspection:] The Github repository for this project will be
+  checked to make sure bugs are listed as issues to keep track of what needs to
+  be done in the system 
+\end{description}
+
+\paragraph{Validate Development Standards}
+\begin{enumerate}
+  \item{test-DS-1}
+  
+  NFR: MS-M3
+
+  Type: Manual, Repository Inspection
+
+  Initial State: Development temporarily paused
+
+  Output/Result: Indication of bugs being tracked on Github
+
+  How test will be performed:
+  \begin{enumerate}
+    \item A tester will look at the 'issues' section of the github repository 
+    \item They will check that there are bugs that exist as issues
+    \item If there aren't, they will consult the team and ask if developers are
+    issuing bugs or if there are simply no bugs
+  \end{enumerate}
+
+
+  \item{test-DS-2}
+  
+  NFR: CR-SC1
+
+  Type: Manual, Code Inspection
+
+  Initial State: Development temporarily paused
+
+  Output/Result: Indication of components not adhering to software development standards
+
+  How test will be performed:
+  \begin{enumerate}
+    \item Team members will regularly inspect the code and ensure all components follow software development standards
+    \item If there are components that don't they will consult the other team members to discuss issues with that component.
+  \end{enumerate}
+\end{enumerate}
+
+\subsubsection{Documentation}
+This section is to ensure that documentation exists to guide users and help them troubleshoot issues they are having.
+
+\begin{description}
+  \item[Documentation Review:] A tester will look through the webpage, and
+  ensure that documentation exists. Then, they must check to make sure the
+  documentation is accurate, and not misleading. 
+\end{description}
+
+\paragraph{Validate Program, help, and training documentation exists and is accurate}
+\begin{enumerate}
+  \item{test-D-1}
+  
+  NFR: MS-S1, UH-L2, OE-P2
```

Comment:
I think MS-S1 should be included in the previous test, it is more so about internal API documentation than user documentation.
```suggestion
  NFR: UH-L2, OE-P2
```

Reply:
@MohammadMohsinKhan FYI